### PR TITLE
Feature:Hello world

### PR DESCRIPTION
**Pull request by yanchuner**
### What?

- brief:这是第二次pr，修改文件结构和图片
- author:Aria

### What changes are included in this PR?

打印输出“Hello world”

### CODE REVIEW

**Comment by Light-City** [comment]
> ### What?
> * brief:这是第一次pr，用于测试
> * author:Aria
> 
> ### What changes are included in this PR?
> 打印输出“Hello world”
> 
> ### Are these changes tested?
> image

img is none.

**Comment by Light-City** [comment]
1.The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

2.pr title is wrong.

File: stl/fight_flow/README.md
Comment (martinx) [inline stl/fight_flow/README.md:1]: Do not remove the other guys' codes.

File: stl/fight_flow/light-city/src/main.cpp
Comment (martinx) [inline stl/fight_flow/light-city/src/main.cpp:1]: Do not remove the other guys' codes.

File: stl/README.md
Comment (martinx) [inline stl/README.md:1]: Do not remove the other guys' codes.

File: stl/study_dayone/wangyanchun/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include<iostream>
+using namespace std;
```
Comment (martinx) [inline stl/study_dayone/wangyanchun/src/main.cpp:2]: You may not use a using-directive to make all names from a namespace available.

File: stl/study_dayone/wangyanchun/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+int main() {
+	std::cout << "hello world" << std::endl;
```
Comment (martinx) [inline stl/study_dayone/wangyanchun/src/main.cpp:5]: Indent 2 spaces.

**Review by martinx — COMMENTED** [review]
https://google.github.io/styleguide/cppguide.html

**Comment by martinx** [comment]
The pr title is still wrong, please refer to others

**Pull request closed without merging**